internal/web_server/web/views/components/quiz_components/article_list.templ

```diff
@@ -6,10 +6,25 @@ import (
 
 // This a component that takes a list of articles and renders them as ArticleCards.
 templ ArticleList(articles *[]articles.Article) {
-	<p class="text-center my-5">Sjekk ut artiklene som spørsmålene var basert på!</p>
-	<div class="flex flex-col gap-4">
-		for _, article := range *articles {
-			@ArticleCard(&article)
-		}
-	</div>
+	<section
+		class="flex flex-col justify-start items-center gap-2 px-1 py-5 h-screen isolate"
+	>
+		<h1 class="font-bold text-3xl">Artikler</h1>
+		<p class="text-center">Sjekk ut artiklene som spørsmålene var basert på!</p>
+		<div class="flex flex-col gap-4 overflow-y-auto scrollbar-hide grow">
+			if len(*articles) == 0 {
+				<p class="text-center">Ingen artikler å vise</p>
```

Comment:
If there are no articles, do we skip this page entirely?

Reply:
1f39f9013f684ab6edcc2eef1212a659ec4b3d99